tests/trainer/test_lr_finder.py

```diff
@@ -179,24 +179,3 @@ def test_suggestion_parameters_work(tmpdir):
 
     assert lr1 != lr2, \
         'Skipping parameter did not influence learning rate'
-
-
-def test_suggestion_with_non_finite_values(tmpdir):
```

Comment:
i didn't see this moved anywhere, why not keep it?

Reply:
not sure why it is missing...